docs/utilities/parameters.md

```diff
@@ -166,6 +168,52 @@ in order to get data from other regions or use specific credentials.
     }
     ```
 
+## DynamoDB 
+To get secrets stored in DynamoDB, use `getDynamoDbProvider`, providing the name of the table that
+contains the secrets. As with the other providers, an overloaded methods allows you to retrieve 
+a `DynamoDbProvider` providing a client if you need to configure it yourself. 
+
+=== "DynamoDbProvider"
+
+    ```java hl_lines="6 9"
+    import software.amazon.lambda.powertools.parameters.DynamoDbProvider;
+    import software.amazon.lambda.powertools.parameters.ParamManager;
+
+    public class AppWithDynamoDbParameters implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+        // Get an instance of the DynamoDbProvider
+        DynamoDbProvider ddbProvider = ParamManager.getDynamoDbProvider("my-parameters-table");
+    
+        // Retrieve a single parameter
+        String value = ddbProvider.get("my-key"); 
+    } 
+    ```
+
+=== "DynamoDbProvider with an explicit region"
```

Comment:
maybe say "custom dynamodb client" rather than "explicit region"

Reply:
I like this better - it makes it clear that it let's you do more than just customize the region. I will update the other param provider comments accordingly.